### ISSUE TITLE
Lenovo T520

### ISSUE THREAD

**Issue by Dmitri-Sintsov** [tluck/Lenovo-T420-Clover#31]
Will it work with T520? The hardware is the same except screen dimensions?

**Comment by tluck** [comment]
yes. will work. the EDID i inject are for 1600x900 but the display overrides i created support 1366x768, 1600x900, and 1920x1080. So maybe eliminate the EDID injection.

**Comment by Dmitri-Sintsov** [comment]
Thanks. I have T520 with 1600x900. I will read the insttuctions later, they seem to be overly complex / not automatized enough at the first sight, but maybe because I never tried it uet.

**Comment by Dmitri-Sintsov** [comment]
Maybe then also add T520 to front page readme.

**Comment by tluck** [comment]
well, the instructions are comprehensive not complex. it boils down to:
1. Putting the EFI bundle on a USB to make an macOS installer
2. installing the OS on the HD
3. Putting the EFI bundle on the HD.
